xcp_d/cli/run.py

```diff
@@ -255,13 +255,28 @@ def get_parser():
     )
 
     g_filter = parser.add_argument_group("Filtering parameters and default value")
-    g_filter.add_argument(
+
+    bandpass_filter_params = g_filter.add_mutually_exclusive_group()
+    bandpass_filter_params.add_argument(
```

Comment:
This parameter has to go first to set the default, I think. It looks like it's "first come, first served" when it comes to defining defaults on the same destination variable. Not really a problem, just a funny idiosyncrasy to keep in mind.